### PR TITLE
[DROOLS-6235] Fix path/packagename management in Windows

### PR DESCRIPTION
**Pull request by gitgabrio**
@danielezonca @kostola  @jiripetrlik 

See https://issues.redhat.com/browse/DROOLS-6235

### CODE REVIEW

**Review by danielezonca — APPROVED** [review]

File: kie-pmml-trusty/kie-pmml-evaluator/kie-pmml-evaluator-assembler/src/main/java/org/kie/pmml/evaluator/assembler/service/PMMLAssemblerService.java [hunk 1]
```diff
@@ -53,6 +53,10 @@
     public static final String PMML_COMPILER_CACHE_KEY = "PMML_COMPILER_CACHE_KEY";
     private static final Logger logger = LoggerFactory.getLogger(PMMLAssemblerService.class);
 
+    private static final String FOLDER_SEPARATOR = "/";
+
+    private static final String WINDOWS_FOLDER_SEPARATOR = "\\";
```
Comment (jiripetrlik) [inline kie-pmml-trusty/kie-pmml-evaluator/kie-pmml-evaluator-assembler/src/main/java/org/kie/pmml/evaluator/assembler/service/PMMLAssemblerService.java:58]: I would prefer to rather use File.separator and make it platform independent.

File: kie-pmml-trusty/kie-pmml-commons/src/test/java/org/kie/pmml/commons/utils/KiePMMLModelUtilsTest.java [hunk 2]
```diff
@@ -38,6 +38,7 @@ public static void setup() {
         packageNameMap.put("a spaced name", "aspacedname");
         packageNameMap.put("AnUpperCasedMame", "anuppercasedmame");
         packageNameMap.put("a_Mixed -name", "amixedname");
+        packageNameMap.put("C:\\w-ind_ow Path", "cwindowpath");
```
Comment (jiripetrlik) [inline kie-pmml-trusty/kie-pmml-commons/src/test/java/org/kie/pmml/commons/utils/KiePMMLModelUtilsTest.java:41]: I'm not sure about this and whether we should create a specific tests for Windows and run it on Linux.

File: kie-pmml-trusty/kie-pmml-evaluator/kie-pmml-evaluator-assembler/src/main/java/org/kie/pmml/evaluator/assembler/service/PMMLAssemblerService.java (hunk 1 above)
Comment (gitgabrio) [reply kie-pmml-trusty/kie-pmml-evaluator/kie-pmml-evaluator-assembler/src/main/java/org/kie/pmml/evaluator/assembler/service/PMMLAssemblerService.java:58]: @jiripetrlik 
I have to follow exactly same logic as in StringUtils to be sure that the twos are consistent

File: kie-pmml-trusty/kie-pmml-commons/src/test/java/org/kie/pmml/commons/utils/KiePMMLModelUtilsTest.java (hunk 2 above)
Comment (gitgabrio) [reply kie-pmml-trusty/kie-pmml-commons/src/test/java/org/kie/pmml/commons/utils/KiePMMLModelUtilsTest.java:41]: @jiripetrlik 
That method verify that a string, formatted (just by chance) as a windows path, is correctly parsed - and that must be verified always, regardless on where the test is actually run.

**Review by jiripetrlik — APPROVED** [review]
@gitgabrio 
Thank you for explanation. I can confirm that this PR is fixing the issue.

**Pull request merged**